### PR TITLE
Improving building

### PR DESCRIPTION
**Pull request by ruslandinov**
Here is PT task for this https://www.pivotaltracker.com/story/show/122031147

What was changed:
- debug build option removed from wrapper & bundle 
- added uglification & minimisation for wrapper & bundle
- added wrapper_dev build (watchify)
- removed outdated code from example pages
- rearranged example pages code for better readability

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,6 +45,7 @@
     "exorcist": "^0.4.0",
```
Comment (tchakabam) [inline package.json]: you can remove the exorcist dep now if you remove usage of it everywhere

**Comment by tchakabam** [comment]
nice! but fix my comment about deps. 

👍

**Comment by ruslandinov** [comment]
Thanks, well noted 👍

**Comment by jlouazel** [comment]
Even if I'm not a big fan of your ruby scripts to launch tasks, all is clear and well organized. Merge this one 🚀

**Pull request merged**